VideoApp/VideoApp/Helpers/Notifications.swift

```diff
@@ -21,6 +21,7 @@ extension Notification.Name {
     static let roomUpdate = Notification.Name("RoomUpdate")
     static let participantsStoreUpdate = Notification.Name("ParticipantsStoreUpdate")
     static let mainParticipantStoreUpdate = Notification.Name("MainParticipantStoreUpdate")
+    static let chatStoreUpdate = Notification.Name("ChatStoreUpdate")
```

Comment:
Because both the room screen and chat screen will need to listen for updates I am using notifications like I did for the video SDK integration.